webplatformcompat/tests/test_tasks.py

```diff
@@ -26,7 +26,7 @@ def test_update_cache_no_invalidation(self):
         self.mock_cache.update_instance.return_value = []
         update_cache_for_instance('Maturity', self.mat.id)
         self.mock_cache.update_instance.assert_called_once_with(
-            'Maturity', self.mat.id, None, None, update_only=True)
+            'Maturity', self.mat.id, None, None, update_only=False)
```

Comment:
The mock is for ``cache.update_instance``, which is being called with explicit parameters - see webplatformcompat/tasks.py